client/helpers/scenes.ts

```diff
@@ -141,33 +144,38 @@ export async function createSceneAndSceneItems({
     sceneId,
   });
   const { a: queuedItems, b: errors } = partition(
-    res.queuedSceneItems,
+    createRes.queuedSceneItems,
     (i: QueuedSceneItem) => isQueuedJob(i.res)
   );
 
-  const queued = returnQueued ? res.queuedSceneItems : [];
-  if (queuedItems.length === 0 || errors.length === res.leaves) {
+  const queued = returnQueued ? createRes.queuedSceneItems : [];
+  if (queuedItems.length === 0 || errors.length === createRes.leaves) {
     return { errors, queued, scene };
   }
 
-  const limit = pLimit(parallelism);
-  await Promise.all(
-    queuedItems.map((is) =>
-      limit<QueuedSceneItem[], void>(async (req: QueuedSceneItem) => {
-        const r = await pollQueuedJob<SceneItem>({
-          id: (req.res as QueuedJob).data.id,
-          getQueuedJob: (id) => client.sceneItems.getQueuedSceneItem({ id }),
-          allow404: true,
-          polling,
-        });
-        if (isPollError(r.res)) {
-          failFast
-            ? throwOnError(r)
-            : errors.push({ req: req.req, res: r.res });
-        }
-      }, is)
-    )
-  );
+  if (verbose) onMsg(`Polling for completed scene-items...`);
+
+  const limit = pLimit(Math.min(parallelism, 20));
+  const cnt = queuedItems.length;
+  const step = Math.floor(cnt / ((PollPercentage / 100) * cnt));
+  const qis = [];
+  // Poll for percentage of items starting at end of `queuedItems` array
+  for (let i = 0; i < cnt; i += step) qis.push(queuedItems[cnt - i - 1]);
```

Comment:
Only poll 10% of `queued-scene-item`s and instead of using the passed `parallelism` for `pLimit`, use a maximum of 20.